pkgs/data/icons/la-capitaine-icon-theme/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "la-capitaine-icon-theme";
+  version = "0.6.1";
+
+  src = fetchFromGitHub {
+    owner = "keeferrourke";
+    repo = "la-capitaine-icon-theme";
+    rev = "v${version}";
+    sha256 = "0cm2scrcg5h45y56h822ywyvfzns1x4wf3gqq96cwb22dc7ny1g9";
+  };
+
+  postPatch = ''
+    rm configure
+  '';
+
+  installPhase = ''
+     mkdir -p $out/share/icons/la-capitaine-icon-theme
+     mv ./* $out/share/icons/la-capitaine-icon-theme
+  '';
+
```

```suggestion

  postInstall = ''
    gtk-update-icon-cache $out/share/icons/la-capitaine-icon-theme
  '';

```

We need to generate the icon cache.
You need to also add `gtk3` as a `nativeBuildInputs`.